main.py

```diff
@@ -758,8 +759,17 @@ def handle_link_down(self, interface):
         if link and link.is_active():
             link.deactivate()
             link.update_metadata('last_status_change', time.time())
+            link.update_metadata('last_status_is_active', False)
```

Comment:
I think to be completely safe we should start also validating and not allowing certain metadata keys to be set by users, like reserved metadata keys or something, otherwise it could lead to potential issues, wdyt?

Reply:
yep for sure, and thanks for reviewing, I've mapped it on this issue https://github.com/kytos-ng/topology/issues/51 @ajoaoff